debuginfo/src/pdb.rs

```diff
@@ -1009,7 +1009,7 @@ impl<'s> Unit<'s> {
 
         let mut functions = Vec::new();
         let mut stack = FunctionStack::new();
-        let mut proc_offsets = SmallVec::<[_; 3]>::new();
+        let mut proc_offsets = SmallVec::<[_; 10]>::new();
```

Comment:
can you add a comment explaining the magic number here?

Reply:
Actually, this change is bogus.